composer.json

```
@@ -48,20 +48,21 @@
   },
   "minimum-stability": "stable",
   "require": {
-    "symfony/console": "^4.0 | ^5.0",
-    "symfony/process": "^4.0 | ^5.0",
-    "php": "^7.2 || ~8.0.0",
+    "php": "^7.4 || ^8.0.0",
```

```suggestion
    "php": "^7.4 || ~8.0.0 || ~8.1.0",
```
php 8.1 and 8.2 and 8.99 is all supported?